vm/map.c

```diff
@@ -673,7 +676,7 @@ static void map_pageFault(unsigned int n, exc_context_t *ctx)
 
 #ifdef PAGEFAULTSTOP
 	process_dumpException(n, ctx);
-	__asm__ volatile ("1: b 1b");
+	__asm__ volatile("1: b 1b");
```

Comment:
clang-format is mistaken

Reply:
About that we are not concitend in the code as sometimes there is space and sometimes there's not that's why i decided to stay with clang-format but I'll change it back